### DISCUSSION TITLE
Why is "Carillon" in "World Music"?

### DISCUSSION THREAD

**Discussion by Fourniture** [musescore/MuseScore#25952] [General]
Currently, the instrument "Carillon" is under the category "World Music":
![image](https://github.com/user-attachments/assets/88d9cab6-9531-400a-b160-b7565ebb4020)
Am I missing something, or should Carillon not belong there?
It certainly does not belong under "Percussion"; it is a keyboard instrument. It seems to have been confused with a Bavarian mechanical clock tower, which is **not** played by a person.
If moved, it would simply go under "All Instruments", because it does not belong to any particular category. (It is, in virtually all cases, a solo instrument.)

**Comment by DaddyLudwig** [comment]
Yeah, Carillon is essentially the keyboard version of tubular bells.

**Reply by Fourniture** [reply]
Pardon my correction, but a Carillon is a tower full of _conventional_ bells, weighing many tons. (The smallest bell is big enough to fit over a person's head.) It is played with both the hands and the feet (like an organ.)

**Comment by Jojo-Schmitz** [comment]
See https://en.wikipedia.org/wiki/Carillon
World Music might be right, but Percussion surely is wrong

**Reply by Fourniture** [reply]
I'll log it as an issue.

**Reply by SteveBlower** [reply]
>Percussion surely is wrong

First sentence of that Wikipedia link: *"A carillon [...]is a pitched percussion instrument"*

Percussion is surely a possibility!

**Reply by Jojo-Schmitz** [reply]
Oops

**Reply by Fourniture** [reply]
It is _technically_ a pitched percussion instrument, but the addition of a keyboard makes it a keyboard instrument--same for the celesta. The celesta is a type of glockenspiel, but it has a keyboard and is classified by that.

**Reply by DaddyLudwig** [reply]
I thought keyboard glockenspiel was something different from celesta. I think I remember hearing somewhere that only 1 or 2 keyboard glockenspiels are in existence. I know Mozart calls for 3 such keyboard glockenspiels in his opera Die Zauberflöte, this being one of those keyboard glockenspiel moments (only the top melody is heard here cause the video is of someone playing it on a regular glockenspiel, but you can just imagine hearing the chords being played by the left hand underneath that melody as Mozart indicates). https://youtu.be/Ix_uCUQLO_A?si=qOnlDIwvki4Y_Occ

**Reply by Fourniture** [reply]
That's a whole other world of technicalities, which is why I didn't mention it. But yes, you are right: keyboard glockenspiel is different from celesta, although the celesta was invented for the _purpose_ of being a _type_ of keyboard glockenspiel.
